fahrschein-spring-boot-starter/README.md

```diff
@@ -201,4 +201,6 @@ For a complete overview of available properties, they type and default value ple
 | `        │   ├── batch-flush-timeout`         | `int`              | in seconds [default value](https://nakadi.io/manual.html#/subscriptions/subscription_id/events_get*batch_flush_timeout) |
 | `        │   ├── stream-timeout`              | `int`              | in seconds [default value](https://nakadi.io/manual.html#/subscriptions/subscription_id/events_get*stream_timeout) |
 | `        │   └── max-uncommitted-events`      | `int`              | [default value](https://nakadi.io/manual.html#/subscriptions/subscription_id/events_get*max_uncommitted_events)            |
-
+| `        └── subscription`                    |                   |                                                                                                                         |
+| `        │   ├── enabled`                     | `boolean`         | `false`                                                                                                                 |
```

Comment:
@coolguy1990 why do we ned a separate section for subscription-by-id and why do we need this enabled property?

Reply:
I can remove the `enabled` part and make it directly `subscription-by-id: String`. My initial plan was to follow other sections such as `oauth`, `http`, `backoff` etc and have an `enabled` flag. 

But if think this `enabled` flag is not required, i can change this to `subscription-by-id` attribute.